requirements.txt

```diff
@@ -7,12 +7,12 @@ tensorflow~=2.8.0  # manually matched to deepcell-tf
 jupyter>=1.0.0,<2
 networkx>=2.1
 opencv-python-headless<5
-deepcell~=0.12.0
+deepcell>=0.12.7
```

Comment:
The new `fetch_data` functionality was added to `deepcell-tf` in 0.12.7, so that's the minimum version required to be able to grab data/models with the deepcell API key. @elaubsch are there any other issues you'd see with bumping the minimum required version of `deepcell` from 0.12 -> 0.12.7?

Reply:
No, I agree that this is a better way to define the requirements! Thanks for catching this.